README.md

```diff
@@ -222,3 +222,8 @@ This project is [MIT](./LICENSE) licensed.
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Deploy
+
+To run my portfolio online, click the link below:[Deploy portfolio.](https://booze33.github.io/My-Portfolio/)
+
```

Comment:
**OPTIONAL** - There is already a section for the deploy header. Please edit that section and include your nav link.